### PR TITLE
kernel: 4.16.12-rt5 -> 4.16.18-rt12

### PR DESCRIPTION
**Pull request by magnetophon**

### CODE REVIEW

**Comment by henrytill** [comment]
Sorry for the latency.

File: modules/base.nix [hunk 1]
```diff
@@ -55,8 +55,6 @@ in {
           else [];
       kernelParams = [ "threadirq" ];
       postBootCommands = ''
```
Comment (henrytill) [inline modules/base.nix]: Seems like you could remove the multiline quotes and just do something like:

```
postBootCommands = optionalString (cfg.soundcardPciId != "") ''
  # other stuff here
'';
```

**Comment by magnetophon** [comment]
OK, done.
And sorry for mangling 2 unrelated commits in 1 PR.

**Comment by henrytill** [comment]
> And sorry for mangling 2 unrelated commits in 1 PR.

No problem.  Thanks for the updates.

**Comment by henrytill** [comment]
We should probably update the `README` now that we have removed those settings.

**Pull request merged**